### PR TITLE
update migrate polling status.

### PR DESCRIPTION
**Pull request by Alice1319**
fixes https://github.com/eclipse-xpanse/xpanse/issues/2086

migrate success:
Snipaste_2024-11-12_18-12-18-migrate-success

migrate request failed:
Snipaste_2024-11-12_16-32-47--migrate-error-pls visit

migrate-when-deploy-failed:
Snipaste_2024-11-12_16-14-03-migrate-deploy-failed-with-errMsg
Snipaste_2024-11-12_16-18-59-migrate-deploy-failed-save-errMsg

migrate-when-destroy-failed:
Snipaste_2024-11-12_16-44-32-migrate-destroy-failed

Snipaste_2024-11-12_16-48-48-myservices-destroy-failed

Snipaste_2024-11-12_16-46-28-destroy-failed-when-migrate-table-save-errMsg

### CODE REVIEW

**Comment by wuzhuanhong** [comment]
LGTM

**Comment by yuanxiaojun987** [comment]
LGTM

File: src/components/content/order/migrate/useMigrateServiceQuery.ts [hunk 1]
```diff
@@ -75,10 +52,33 @@ export function useServiceDetailsPollingQuery(
...
-            (currentMigrationStatus === migrationStatus.MIGRATION_COMPLETED ||
-                currentMigrationStatus === migrationStatus.DEPLOY_FAILED ||
-                currentMigrationStatus === migrationStatus.DESTROY_FAILED),
+            (currentMigrationTaskStatus === taskStatus.SUCCESSFUL || currentMigrationTaskStatus === taskStatus.FAILED),
         staleTime: Infinity,
         gcTime: Infinity,
     });
 }
+
+export function useLatestServiceOrderStatusQuery(
```
Comment (swaroopar) [inline src/components/content/order/migrate/useMigrateServiceQuery.ts:61]: there is already one useLatestServiceOrderStatusQuery in useServiceDetailsPollingQuery.ts
We must reuse it. 
if possible you can move useLatestServiceOrderStatusQuery out of useServiceDetailsPollingQuery.ts to another file and use it from everywhere.

**Review by swaroopar — CHANGES_REQUESTED** [review]

File: src/components/content/order/migrate/useMigrateServiceQuery.ts (hunk 1 above)
Comment (Alice1319) [reply src/components/content/order/migrate/useMigrateServiceQuery.ts:61]: good idea, let me udate this

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**